lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -388,11 +388,21 @@ function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, ca
                                 return;
                             }
                         }
-
                         options.json = {
                             actionType: 'append',
                             entities: result
                         };
+                        if (
+                            'autoprovision' in typeInformation &&
+                            /* jshint -W101 */
+                            typeInformation.autoprovision !== undefined
+                                ? typeInformation.autoprovision === true
+                                : config.getConfig().appendMode === true
+                        ) {
+                            options.json.actionType = 'append';
+                        } else {
+                            options.json.actionType = 'update';
+                        }
                     } else {
```

Comment:
Apply fix to these else branch needs to update 81 tests!
https://github.com/telefonicaid/iotagent-node-lib/pull/1061
https://github.com/telefonicaid/iotagent-node-lib/pull/1060